portal-api/src/v1/gef/controllers/validation/facilities.js

```diff
@@ -21,6 +21,16 @@ const hasRequiredItems = (doc) => {
   if (doc.hasBeenIssued === true && !doc.coverEndDate) {
     required.push('coverEndDate');
   }
+  // TODO DTFS2-7171: Add logic to only push the below to required if the deal version exists and is > 1
```

Comment:
Same comment -- could we do this as part of this ticket and rebase ontop of Marks?

Reply:
Done :)